<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udpComm</class>
 <widget class="QMainWindow" name="udpComm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP通信程序--测试版</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>源机器端口：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxbindport">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>1567</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>目标IP地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxtargetip"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>目标端口号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxtargetport">
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>1568</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="text">
            <string>启动服务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop">
           <property name="text">
            <string>停止服务</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEditmsg"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sendmsg">
           <property name="text">
            <string>发送消息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_broadcastmsg">
           <property name="text">
            <string>广播消息</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEditdispmsg"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
